Cargo.toml

```diff
@@ -36,6 +36,8 @@ num_cpus = "^1.0.0"
 zopfli = "^0.4.0"
 miniz_oxide = "0.3"
 rgb = "0.8.11"
+vec_map = "0.8.1"
```

Comment:
I don't see this dependency used anywhere. Can it be removed?

Reply:
Oops, yes, that was a temporary experiment. Good catch.